config/manager/kustomization.yaml

```diff
@@ -4,5 +4,10 @@ kind: Kustomization
 resources:
   - manager.yaml
 
+configMapGenerator:
+- name: fast-track-configmap
+  literals:
+  - CAPM3_FAST_TRACK=${CAPM3_FAST_TRACK}
```

Comment:
you should default to false here to keep the same behaviour as before if it is unset